src/main/java/com/gachigang/ontherun/controller/CategoryController.java

```diff
@@ -0,0 +1,51 @@
+package com.gachigang.ontherun.controller;
+
+import com.gachigang.ontherun.common.mapper.CategoryMapper;
+import com.gachigang.ontherun.payload.category.request.CreateCategoryRequest;
+import com.gachigang.ontherun.payload.category.request.UpdateCategoryRequest;
+import com.gachigang.ontherun.payload.category.response.CategoryResponse;
+import com.gachigang.ontherun.service.CategoryService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@Slf4j
+@RequestMapping("/category")
+public class CategoryController {
+
+    private final CategoryService categoryService;
+    private final CategoryMapper categoryMapper;
+
+    @GetMapping
+    public ResponseEntity <List<CategoryResponse>> findAllCategory(
+            @RequestParam(required = false, defaultValue = "1") int pageNumber,
```

Comment:
Remove pagination and extra space after ResponseEntity

Reply:
Done